en/flight_modes/mission.md

```diff
@@ -162,3 +166,27 @@ Vehicles switch to the next waypoint as soon as they enter the acceptance radius
   - The equation is: 
     $$L_{1_{distance}}=\frac{1}{\pi}L_{1_{damping}}L_{1_{period}}\left \| \vec{v}_{ {xy}_{ground} } \right \|$$
 
+
+## Fixed-wing Mission Takeoff/Landing
+
+Starting and ending flights with mission takeoff and landing is the recommended way of operating a plane autonomously.
+A detailed description of the executed logic during autonomous takeoffs and landings can be found in [Takeoff mode > Fixed-wing](../flight_modes/takeoff.md#fixed-wing-fw) and [Land mode > Fixed-wing](../flight_modes/land.md#fixed-wing-fw).
+The section here is to highlight differences in mission takeoffs and landings compared to the stand-alone flight modes.
+
+### Takeoff
+
+Fixed-wing mission takeoffs can be configured by placing a Takeoff mission item, including an altitude setpoint, on the map.
+The vehicle will then takeoff towards this waypoint, and climb until the altitude specified is reached.
+After reaching the takeoff altitude, the mission will be automatically proceeded with the next waypoint.
+The configuration is the same for wheeled vehicles that do a runway takeoff, or hand-launched ones.
+In both cases, the vehicle should be placed facing the takeoff waypoint when the mission is started, 
+which will arm the vehicle and throttle up the motors.
+
+If possible, always make the vehicle takeoff into the wind.
+
+### Land
+
+Currently the only way to land a vehicle autonomously is through a mission landing.

```

Comment:
Is this true? That's not what it says here: http://docs.px4.io/main/en/flight_modes/land.html#fixed-wing-fw

Reply:
Working on a fix, we should remove this line (or once the fix is in).

Reply:
Do you have a PR for this fix?